src/core/components/button/index.tsx

```diff
@@ -101,9 +101,10 @@ const Button = ({
 	children,
 	...props
 }: ButtonProps) => {
-	const buttonContents = [children]
+	const buttonContents = [<div className="baseline-kick">{children}</div>]
```

Comment:
We'll need to apply these changes to [LinkButton](https://github.com/guardian/source/blob/master/src/core/components/button/index.tsx#L159) too. I'll look into consolidating these components when the API is mostly aligned.